app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::SessionsController < Devise::SessionsController
+  def after_sign_in_path_for(resource)
+    developer_path resource
```

Comment:
If the user doesn't have a developer profile yet then `resource` is nil and this raises an exception.

Reply:
Solved by checking if the developer is present

```ruby
  def after_sign_in_path_for(resource)
    if resource.developer.present?
      developer_path resource.developer
    else
      root_path
    end
  end
```